### ISSUE TITLE
Problem retrieving results for https://www.bbc.com

### ISSUE THREAD

**Issue by LarryKHite** [jlhernando/google-pagespeed-bulk#1]
Problem retrieving results for https://www.bbc.com
undefined
Writing lab data...
(node:16268) UnhandledPromiseRejectionWarning: Error: Data should not be empty or the "fields" option should be included       
    at JSON2CSVParser.preprocessData (D:\Code\google-pagespeed-bulk\node_modules\json2csv\lib\JSON2CSVParser.js:57:13)
    at JSON2CSVParser.parse (D:\Code\google-pagespeed-bulk\node_modules\json2csv\lib\JSON2CSVParser.js:20:32)
    at module.exports.parse (D:\Code\google-pagespeed-bulk\node_modules\json2csv\lib\json2csv.js:15:65)
    at getSpeedData (file:///D:/Code/google-pagespeed-bulk/index.js:172:45)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:16268) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of 
an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:16268) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that 
are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by jlhernando** [comment]
Hi @LarryKHite is that he only url that you have in the 'urls.csv' file? Could you double check that you have added an API Key in the 'api-request.js' file.

**Comment by LarryKHite** [comment]
i'am sure API Key is correct, No matter one url or multiple urls all are wrong😭

> Hi @LarryKHite is that he only url that you have in the 'urls.csv' file? Could you double check that you have added an API Key in the 'api-request.js' file.

**Comment by jlhernando** [comment]
That's very strange. The only way I've been able to get "undefined" is where there is a connection error so thanks for flagging this as I didn't account for it in the initial commit.

![undefined-response](https://user-images.githubusercontent.com/14793651/118664617-874a7980-b7e9-11eb-84ab-b49971eaa9a2.JPG)

I've modified the script to account for when that happens to get a message logged. I'll upload the new version soon.

However, I find odd that you get a connection error for multiple URLs. Are you using a VPN? or could your connection be very unstable?
